setup.py

```diff
@@ -19,17 +19,17 @@
 
 def readme():
     """Read in and return the contents of the project's README.md file."""
-    with open("README.md", encoding="utf-8") as f:
-        return f.read()
+    with open("README.md", encoding="utf-8") as in_file:
+        return in_file.read()
```

Comment:
This change from the skeleton is unnecessary and should be reverted:
```suggestion
    with open("README.md", encoding="utf-8") as f:
        return f.read()
```